### DISCUSSION TITLE
decoding QUIACHIP TX118SA-4 ?

### DISCUSSION THREAD

**Discussion by grilli303** [merbanan/rtl_433#2450] [Q&A]
Hi There
First of all thanks to the developers of this great software.
I would be interested to know if it is possible to decode the signals of the QUIACHIP TX118SA-4?
I couldn't find anything on the www related to this topic.

Modulation: ASK
Encoding: Learning code EV1527

https://qiachip.com/blogs/usermenu/qiachip-tx118sa-transmitter-module-instructions-manual-wireless-433-92mhz-rf-module-1527-learning-code-superheterodyne-receiver-wireless-decoding

Has anyone tried to see if it works?

Kind regards, Daniel

**Comment by zuckschwerdt** [comment]
Yes, EV1527 is very easy and will always work, see https://github.com/merbanan/rtl_433/tree/master/conf for examples.

**Reply by grilli303** [reply]
Thanks a lot for this very quick reply.👍
I will check it out...
Kind regards and greetings from Switzerland.
Daniel
